scripts/dotnet.py

```diff
@@ -919,25 +916,11 @@ def add_arguments(parser: ArgumentParser) -> ArgumentParser:
     '''
     Adds new arguments to the specified ArgumentParser object.
     '''
-
     parser = __add_arguments(parser)
-
-    # .NET Compilation modes.
```

Comment:
Do we just no longer need this argument?

Reply:
Yep, I checked and this argument was not used anywhere